docker/tif-metadata/Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM alpine
+
+RUN apk update
+RUN apk add bash python3
+RUN pip3 install boto3 docopt awscli
```

Comment:
Suggest you move this into a requirements.txt where we can pin the version, then:

`pip3 install -r requirements.txt`